### PR TITLE
Asb staging

### PR DESCRIPTION
**Pull request by austinclprojects**

### CODE REVIEW

File: scripts/settings/default/main.lua [hunk 1]
```diff
@@ -83,7 +83,7 @@ xi.settings =
     NORMAL_MOB_MAX_LEVEL_RANGE_MAX = 0,  -- Upper Bound of Max Level Range for Normal Mobs (0 = Uncapped)
     START_GIL                      = 10, -- Amount of gil given to newly created characters.
     START_INVENTORY                = 30, -- Starting inventory and satchel size.  Ignores values < 30.  Do not set above 80!
-    NEW_CHARACTER_CUTSCENE         = 1,  -- Set to 1 to enable opening cutscenes, 0 to disable.
+    NEW_CHARACTER_CUTSCENE         = 0,  -- Set to 1 to enable opening cutscenes, 0 to disable.
```
Comment (kaincenteno) [inline scripts/settings/default/main.lua:86]: why is this being disable as the default?

File: sql/zone_settings.sql [hunk 2]
```diff
@@ -39,305 +39,305 @@ CREATE TABLE `zone_settings` (
...
+INSERT INTO `zone_settings` VALUES (289,2,'149.56.148.173',54230,'Escha_RuAun',80,80,80,80,0,0.00,2200);
+INSERT INTO `zone_settings` VALUES (290,0,'149.56.148.173',54230,'Desuetia_Empyreal_Paradox',0,0,0,0,0,0.00,2176);
+INSERT INTO `zone_settings` VALUES (291,0,'149.56.148.173',54230,'Reisenjima',79,79,79,79,0,0.00,2176);
+INSERT INTO `zone_settings` VALUES (292,0,'149.56.148.173',54230,'Reisenjima_Henge',0,0,0,0,0,0.00,2048);
+INSERT INTO `zone_settings` VALUES (293,0,'149.56.148.173',54230,'Reisenjima_Sanctorium',0,0,0,0,0,0.00,2176);
+INSERT INTO `zone_settings` VALUES (294,5,'149.56.148.173',54230,'Dynamis-San_dOria_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (295,5,'149.56.148.173',54230,'Dynamis-Bastok_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (296,5,'149.56.148.173',54230,'Dynamis-Windurst_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (297,5,'149.56.148.173',54230,'Dynamis-Jeuno_[D]',88,88,88,88,0,0.00,2448);
+INSERT INTO `zone_settings` VALUES (298,0,'149.56.148.173',54230,'Walk_of_Echoes_[P1]',186,186,186,186,0,0.00,2048);
```
Comment (kaincenteno) [inline sql/zone_settings.sql:340]: why the change away from localhost?

**Comment by TeoTwawki** [comment]
> Merges new changes from LSB and ASB into base.

That is not at all useful as a description. All in one gigantic merges are also a big "nope". This looks mor elike a mistake where you intended to send this elsewhere instead of here.

File: scripts/settings/default/main.lua (hunk 1 above)
Comment (austinclprojects) [reply scripts/settings/default/main.lua:86]: this was opened in error, my bad.

File: sql/zone_settings.sql (hunk 2 above)
Comment (TeoTwawki) [reply sql/zone_settings.sql:340]: probably he did not intend to PR from his fork into LSB. I'm thinking he meant to send all this up to his fork.

Comment (austinclprojects) [reply sql/zone_settings.sql:340]: Yea meant to merge into my fork, didn't notice it was PRing upstream til too late.
